cppcache/src/CacheXmlParser.cpp

```diff
@@ -562,27 +555,15 @@ void CacheXmlParser::startCache(const xercesc::Attributes &attrs) {
     }
   }
 
-  _GEODE_NEW(cacheCreation_, CacheXmlCreation());
+  cacheCreation_ = make_unique<CacheXmlCreation>();
 }
 
 void CacheXmlParser::startPdx(const xercesc::Attributes &attrs) {
-  auto ignoreUnreadFields = getOptionalAttribute(attrs, IGNORE_UNREAD_FIELDS);
-  if (!ignoreUnreadFields.empty()) {
-    if (equal_ignore_case(ignoreUnreadFields, "true")) {
-      cacheCreation_->setPdxIgnoreUnreadField(true);
-    } else {
-      cacheCreation_->setPdxIgnoreUnreadField(false);
-    }
-  }
-
-  auto pdxReadSerialized = getOptionalAttribute(attrs, READ_SERIALIZED);
-  if (!pdxReadSerialized.empty()) {
-    if (equal_ignore_case(pdxReadSerialized, "true")) {
-      cacheCreation_->setPdxReadSerialized(true);
-    } else {
-      cacheCreation_->setPdxReadSerialized(false);
-    }
-  }
+  CacheRegionHelper::getCacheImpl(cache_)->setPdxIgnoreUnreadFields(
```

Comment:
Are you certain this logic does not override the default value of property. The original code only change the value if the values was provided in the XML, preserving the default value in the object.